src/sagemaker/accept_types.py

```diff
@@ -77,6 +77,7 @@ def retrieve_default(
     tolerate_deprecated_model: bool = False,
     sagemaker_session: Session = DEFAULT_JUMPSTART_SAGEMAKER_SESSION,
     model_type: JumpStartModelType = JumpStartModelType.OPEN_WEIGHTS,
+    config_name: Optional[str] = None,
```

Comment:
in the future, we should think of a way to consolidate all these JS related fields perhaps into a dataclass or kwargs, so we don't need to update all these function prototypes whenever a new feature is added.